package.json

```
@@ -32,13 +37,15 @@
     "eslint-plugin-import": "^2.29.1",
     "lintspaces-cli": "^0.6.0",
     "pa11y-ci": "^1.2.0",
+    "react-dom": "^18.3.1",
     "sass": "^1.51.0",
     "scss-lint": "0.0.0",
+    "sucrase": "^3.35.0",
     "typescript": "^5.5.3"
   },
   "scripts": {
-    "demo": "npm run demo-build && node demos/app",
-    "demo-build": "npm run build && sass demos/src/demo.scss public/main.css --load-path node_modules",
+    "demo-start": "node demo/app.js",
```

```suggestion
    "demo": "node demo/app.js",
```

Not super important but I think it's easier to just write and understand `npm run demo`